### PR TITLE
Improve the guest setup setup

### PR DESCRIPTION
**Pull request by hector-cao**
Create common funtion to deploy the tests code to the guest and run the guest setup script to properly setup the guest before the test

### CODE REVIEW

File: tests/tests/test_guest_measurement.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 import Qemu
 import util
+from common import *
 
 script_path=os.path.dirname(os.path.realpath(__file__))
```
Comment (spmcmillan) [inline tests/tests/test_guest_measurement.py:27]: same here

File: tests/tests/test_guest_report.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 import Qemu
 import util
+from common import *
 
 script_path=os.path.dirname(os.path.realpath(__file__))
```
Comment (spmcmillan) [inline tests/tests/test_guest_report.py:27]: same here

File: tests/tests/test_quote_configfs_tsm.py [hunk 3]
```diff
@@ -20,6 +20,7 @@
 
 import Qemu
 import util
+from common import *
 
 script_path=os.path.dirname(os.path.realpath(__file__))
```
Comment (spmcmillan) [inline tests/tests/test_quote_configfs_tsm.py:25]: same here

File: tests/setup_guest.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# this script is supposed to be executed under root
+
+apt install -y python3 python3-pip
```
[the remark this answers is not in the document]
Comment (hector-cao) [reply tests/setup_guest.sh:5]: @spmcmillan  You are suggesting to put this setup part in the image creation but by putting it here, i was aiming to eliminate a prerequisite the testing has on the image creation process. We can add this setup on both (image creation and tests setup) but at least, i would like to have this in tests setup phase.

File: tests/tests/test_boot_basic.py [hunk 5]
```diff
@@ -17,6 +17,7 @@
 
 import os
 import Qemu
+from common import *
 
 script_path=os.path.dirname(os.path.realpath(__file__))
```
[the remark this answers is not in the document]
Comment (hector-cao) [reply tests/tests/test_boot_basic.py:22]: i think this is necessary to have the symbol `deploy_and_setup` ? isn't it ?

[the remark this answers is not in the document]
Comment (spmcmillan) [reply tests/tests/test_boot_basic.py:22]: I meant to point at the "script_path=" line

[the remark this answers is not in the document]
Comment (hector-cao) [reply tests/tests/test_boot_basic.py:22]: ah sorry, i read it too quick

**Review by spmcmillan — APPROVED** [review]
LGTM

**Pull request merged**